src/main/scala/io/github/kiemlicz/shelm/HelmPlugin.scala

```diff
@@ -76,13 +79,23 @@ object HelmPlugin extends AutoPlugin {
             mappings.settings.chartLocation,
             target.value / s"${mappings.settings.chartLocation.chartName.name}-$idx", log
           )
-          if (mappings.dependencyUpdate) {
-            updateDependencies(tempChartDir, log)
-            (tempChartDir ** "*.tgz").get()
-              .foreach { f =>
-                ChartDownloader.extractArchive(f.toURI, tempChartDir / DependenciesPath)
-                IO.delete(f)
+          mappings.dependencyUpdate match {
```

Comment:
Ok, my initial thought was: why do you add this logic since we have: `io.github.kiemlicz.shelm.ChartMappings#chartUpdate` which can transform your dependencies, but it is executed after `dependency update`  
So the question is: shouldn't the Chart.yaml modification happen first, write the modification to the Chart.yaml and then the `dependency update` can be performed?

The risk: the dependency update may fetch changed dependencies, never fetching original ones, but isn't it desired behavior?

Reply:
Yeah, this would be better, will change.

I was thinking that such action ordering change would be unwelcome if you were allowed to programatically change contents of fetched dependend charts in `chartUpdate`, but this is not allowed, and probably will not be, so no problem there

Reply:
given the user knows where the unpacked dependencies will go, it will still be able to override them (if proper `includeFiles` are used)  
However I think we can proceed with this change